Tasks/VsTestV2/nondistributedtest.ts

```diff
@@ -39,7 +39,7 @@ export class NonDistributedTest {
             }
 
         } catch (err) {
-            tl.setResult(tl.TaskResult.Failed, tl.loc('VstestFailedReturnCode'), true);
+            tl.setResult(tl.TaskResult.Failed, err, true);
```

Comment:
Can we double check whether, we are logging the stack trace of the error, or just the error statement?